__tests__/app.test.js

```diff
@@ -35,13 +35,28 @@ describe('GET /api', () => {
 })
 
 describe('GET api/articles/:article_id', () => {
-    test('resolves with a status 200 and the correct article and properties', () => {
+    test('resolves with a status 200 and the correct article keys and values', () => {
+        return request(app).get('/api/articles/1').expect(200).then((res) => {
+            const article = res.body.article
+            expect(article).toEqual(expect.objectContaining({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 100,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                comment_count: 11
+            }))
```

Comment:
The strictly TDD approach would have been to keep your original tests, and add a new test that only expects the object to contain `comment_count` as one of it's properties